src/explore-education-statistics-frontend/.env

```diff
@@ -5,4 +5,5 @@ PUBLIC_API_DOCS_URL=TODO-GUIDANCE-URL
 NOTIFICATION_API_BASE_URL=http://localhost:7073/api
 GA_TRACKING_ID=
 PUBLIC_URL=http://localhost:3000/
+PROD_PUBLIC_URL=https://explore-education-statistics.service.gov.uk/
```

Comment:
Could get rid of the trailing slash given? We don't need to change this in all the environments or infrastructure so this should be pretty easy to do.

Will need adjustments in the `next-sitemap.config.js`

Reply:
Hah, yeah happily. I went with the slash to be consistent with PUBLIC_URL but yes would much prefer without